Gemfile.lock

```diff
@@ -200,13 +205,13 @@ GEM
       rspec-core (~> 2.14.0)
       rspec-expectations (~> 2.14.0)
       rspec-mocks (~> 2.14.0)
-    rubocop (0.27.1)
+    rubocop (0.30.0)
```

Comment:
Can we avoid updating rubocop in this PR, or is this version a requirement?  Basically, just reset your Gemfile.lock, and then run bundle update haml-lint and it should update the minimal.